core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/dto/ClaimConfig.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.user.core.dto;
+
+import org.wso2.carbon.user.core.claim.ClaimMapping;
+
+import java.util.Map;
+
+public class ClaimConfig {
```

Comment:
ClaimConfig might perhaps be a critical entity in wherever it is used. Therefore, consider adding a proper class level comment explaining the semantics associated with it.